app/models/ability.rb

```diff
@@ -10,6 +10,7 @@ def initialize(user)
 
     admin_user_permissions(user) if user.role_admin?
     checkin_checkout_user_permissions(user) if user.role_checkin_checkout?
+    checkin_checkout_user_admin_permissions(user) if user.role_checkin_checkout_admin?
```

Comment:
maybe admin is a bit misleading? can we cal it `checkin_checkout_manager?`?

Reply:
totally agree! changed